keysign/bluetoothoffer.py

```diff
@@ -76,29 +76,31 @@ def start(self):
         returnValue((success, message))
 
     def allocate_code(self):
+        """Allocate a BT code only if there is a powered on BT adapter"""
```

Comment:
otherwise what?  Return none or raise?
Also: what does "allocate" mean?

How about something like

Acquires and returns a string suitable for finding the key via Bluetooth. Returns none if no powered on adapter could be found.

Reply:
Yeap it wasn't very clear. Fixed with the last commit.